coil-sample/src/main/java/coil/sample/Extensions.kt

```diff
@@ -37,9 +38,7 @@ fun Context.getDisplaySize(): PixelSize {
     return resources.displayMetrics.run { PixelSize(widthPixels, heightPixels) }
 }
 
-fun Int.dp(context: Context): Float {
-    return this * context.resources.displayMetrics.density
-}
+val Int.dp: Float get() = this * Resources.getSystem().displayMetrics.density
```

Comment:
`Resources.getSystem()` returns the default `Resources` for the app instead of `Context`-specific assets/dimensions for a display and should be avoided.

Reply:
Right, modifed the context's resources can lead `displayMetrics.density` different, reverted.